app/js/views/services-list-item.jsx

```diff
@@ -41,6 +49,14 @@ export default class ServicesListItem extends React.Component {
     }
   }
 
+  componentWillUnmount() {
+    switch (this.service.type) {
+      case 'motion-sensor':
+        this.service.off('motion', this.onMotion);
```

Comment:
I checked and `offAll` already does this if no parameters are supplied. So what about just `this.service.offAll()` here?